### ISSUE TITLE
Error: Aesthetics must be either length 1 or the same as the data (100): x, y, fill

### ISSUE THREAD

**Issue by rickwargo** [boxuancui/DataExplorer#17]
Worked beautifully on the sample data sets. I tried on a personal data set and received the following error:

```
  |......................................                           |  59%
label: density_continuous
  |..........................................                       |  65%
  ordinary text without R code

  |..............................................                   |  71%
label: correlation_continuous
Quitting from lines 51-52 (report.rmd) 
Error: Aesthetics must be either length 1 or the same as the data (100): x, y, fill
```

**Comment by boxuancui** [comment]
@rickwargo,
Could you run `CorrelationContinuous(data)`? Does it throw any error?
Could you also generate some data that could reproduce this bug? Feel free to send me your data if you are OK.

**Comment by rickwargo** [comment]
It appears there may be a conflict with the reshape package. The following steps reproduce the problem. I also get the problem when running CorrelationContinuous.

> library(DataExplorer)
> CorrelationContinuous(mtcars)
> library(reshape)
> CorrelationContinuous(mtcars)
> Error: Aesthetics must be either length 1 or the same as the data (121): x, y, fill

All the best,
Rick Wargo
https://linkedin.com/in/rickwargo/
https://www.rickwargo.com/

**Comment by rickwargo** [comment]
The following set of data generates this error message:

> head(data)
>   mso_meet_id mso_gymnast_id meet_id year event_type           gymnast          team session level  division floor pommel rings vault pbars hibar bars beam
> 1       10101         123260     227 2005        MEN Nickolas Carrillo     Technique       6     7    11 yrs  8.30    6.2  8.30  9.00   7.7   6.9   NA   NA
> 2       10101         123371     227 2005        MEN  Adrian Fernandez San Francisco       7     8 12-14 yrs  7.95    5.3  8.30  7.35   7.5   8.0   NA   NA
> 3       10101         123477     227 2005        MEN    Grayson Huston   Rising Star       6     7   12+ yrs  9.20    8.2  9.20  9.50   8.1   8.6   NA   NA
> 4       10101         123478     227 2005        MEN  Tyler Brady-Riso   Rising Star       6     7  9-10 yrs  7.50    7.0  9.10  8.95   8.2   8.4   NA   NA
> 5       10101         123565     227 2005        MEN   Russell Jegglie     Peninsula       6     7    11 yrs  7.90    6.4  8.75  9.60   8.4   8.0   NA   NA
> 6       10101         123837     227 2005        MEN        Justin Nix       Stellar      3B     5    Middle  7.70    4.1  7.60  8.40   6.3   7.6   NA   NA

Works, with warnings:

> CorrelationContinuous(data6)
> Warning messages:
> 1: In cor(continuous, ...) : the standard deviation is zero
> 2: Removed 90 rows containing missing values (geom_text).

Errors (generates same error with full data set), reshape library not loaded:

> data6 <- head(data)
> GenerateReport(data6)

processing file: report.rmd
  |....                                                             |   6%
  ordinary text without R code

  |........                                                         |  12%
label: global_options (with options)
List of 1
 $ include: logi FALSE

  |...........                                                      |  18%
   inline R code fragments

  |...............                                                  |  24%
label: data_structure
  |...................                                              |  29%
  ordinary text without R code

  |.......................                                          |  35%
label: plot_missing
  |...........................                                      |  41%
  ordinary text without R code

  |...............................                                  |  47%
label: histogram_continuous
  |..................................                               |  53%
  ordinary text without R code

  |......................................                           |  59%
label: density_continuous
  |..........................................                       |  65%
  ordinary text without R code

  |..............................................                   |  71%
label: correlation_continuous
Quitting from lines 51-52 (report.rmd)
Error in seq.default(from = best$lmin, to = best$lmax, by = best$lstep) :
  'from' must be of length 1

All the best,
Rick Wargo
https://linkedin.com/in/rickwargo/
https://www.rickwargo.com/

**Comment by boxuancui** [comment]
This issue is already fixed in issue #18 . Please use the following code to get the latest version.

```
library(devtools)
install_github("boxuancui/DataExplorer", ref="develop")
```

Please re-open if you see same problem.
